WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -262,14 +263,14 @@ + (NSArray *)routes
 
 + (id<FBResponsePayload>)handleGetUIAElementValue:(FBRouteRequest *)request
 {
-  FBElementCache *elementCache = request.session.elementCache;
-  XCUIElement *element = [elementCache elementForIndex:[request.parameters[@"id"] integerValue]];
-  NSString *value = request.arguments[@"value"];
-  if (!value) {
-    return FBResponseWithErrorMessage(@"Missing value parameter");
-  }
-  [element adjustToPickerWheelValue:value];
-  return FBResponseWithOK();
+    FBElementCache *elementCache = request.session.elementCache;
+    XCUIElement *element = [elementCache elementForIndex:[request.parameters[@"id"] integerValue]];
+    if (element.elementType == XCUIElementTypePickerWheel && request.arguments[@"value"]) {
+        NSString *wheelPickerValue = request.arguments[@"value"];
```

Comment:
@marekcirkos i do check for nil:

```
if (element.elementType == XCUIElementTypePickerWheel && request.arguments[@"value"])
```
if request.arguments[@"value"] is nil it wont execute the if block.